quiz/web/src/modules/genericTests/FakerTest.tsx

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { faker } from "@faker-js/faker";
+import type { SexType } from "@faker-js/faker";
+
+type SubscriptionTier = "free" | "basic" | "pro";
+
+class User {
```

Comment:
W TS to jest niepotrzebne, można użyć private albo protected, kod się skróci 3x, a będzie robił to samo. https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd

Reply:
Poprawione, z użyciem public, ponieważ na private lub protected sypało błędami TS